docs/SECURITY.md

```
@@ -185,13 +185,43 @@ CISO エージェントによるメタレビューで「Phase 1 既知制約の
 
 #### Linux 診断手段
 
-`bwrap` 起動が失敗する場合、以下のログで原因を切り分ける:
+`vibecorp-sandbox` の Linux ブランチは bwrap 起動失敗を 4 区分（A/B/C/D）に分けて日本語メッセージで通知する（Issue #579）。`vibecorp-sandbox` の stderr に出力された区分判定をまず確認し、該当する区分の対処手順を参照する。
+
+| 区分 | stderr 抜粋 | 原因 | 対処手順 |
+|------|------------|------|---------|
+| **A** | `bwrap (bubblewrap) が見つかりません` | バイナリ不在 | distro 別 install: `sudo apt-get install bubblewrap` / `sudo dnf install bubblewrap` / `sudo apk add bubblewrap` |
+| **B** | `bwrap バイナリは存在しますが ` `--version` `で起動失敗しました` | バイナリ破損 / 権限不足 | distro 別 reinstall: `sudo apt-get install --reinstall bubblewrap` / `sudo dnf reinstall bubblewrap` / `sudo apk add --upgrade bubblewrap`<br>権限確認: `ls -lah "$(command -v bwrap)"` |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)